packages/sdk/src/create/create.ts

```diff
@@ -10,7 +10,13 @@ import { createLibp2pAndUpdateOptions } from "./libp2p.js";
  * Uses Waku Filter V2 by default.
  */
 export async function createLightNode(
-  options: CreateNodeOptions = {}
+  options: CreateNodeOptions = {
```

Comment:
Let's not add it here and instead set it in packages/sdk/src/create/discovery.ts

Reply:
22df03eefaf093c0330dddbadcd58e644e18cf73